src/pudl/transform/ferc1.py

```diff
@@ -1851,15 +1851,15 @@ def apply_xbrl_calculation_fixes(
                         "calc_component_to_replace": {
                             "name": "nuclear_fuel_materials_and_assemblies",
                             "weight": 1.0,
-                            "source_tables": ["balance_sheet_assets_ferc1"],
+                            "source_tables": [],
                         },
                         "calc_component_new": {},
                     },
                     {
                         "calc_component_to_replace": {
                             "name": "spent_nuclear_fuel",
```

Comment:
Is this actually meant to be `spent_nuclear_fuel_major_only` which also comes from `f1_comp_balance_db`? Possibly OOS for this issue.